src/Core/Service/Subscriber/WebhookManagerSubscriber.php

```diff
@@ -0,0 +1,66 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Service\Subscriber;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Update\Event\UpdatePostFinishEvent;
+use Shopware\Core\Framework\Webhook\Event\PreWebhooksDispatch;
+use Shopware\Core\Framework\Webhook\Webhook;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * @internal
+ */
+#[Package('framework')]
+class WebhookManagerSubscriber implements EventSubscriberInterface
+{
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            PreWebhooksDispatch::class => 'filterDuplicates',
+        ];
+    }
+
+    public function filterDuplicates(PreWebhooksDispatch $event): void
```

Comment:
since this lives in the 'service' bundle, should we also only do the dedup only on the 'self-managed' apps? 🤔

Reply:
yeah that makes sense 👍